### PR TITLE
Delete :intellij Gradle subproject

### PR DESCRIPTION
**Pull request by rli**
It is no longer a relevant configuration after the plugin split in the 3.0 release
 
## License
I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -34,9 +34,12 @@ reported the issue. Please try to include as much information as you can. Detail
 
 ### Instructions
 
-1. Clone the github repository and run `./gradlew :plugin-toolkit:intellij:buildPlugin` <br/> (This will produce a plugin zip under `plugins/toolkit/intellij/build/distributions`)
-2. In your JetBrains IDE (e.g. IntelliJ) navigate to the `Plugins` preferences and select "Install Plugin from Disk...", navigate to the zip file produced in step 1. 
-3. You will be prompted to restart your IDE.
+1. Clone the github repository.
+1. To manually build a plugin distribution, run the `buildPlugin` task on the relevant Gradle project.
```
Comment (wularr) [inline CONTRIBUTING.md:38]:
```suggestion
2. To manually build a plugin distribution, run the `buildPlugin` task on the relevant Gradle project. 
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -34,9 +34,12 @@ reported the issue. Please try to include as much information as you can. Detail
...
 ### Instructions
 
-1. Clone the github repository and run `./gradlew :plugin-toolkit:intellij:buildPlugin` <br/> (This will produce a plugin zip under `plugins/toolkit/intellij/build/distributions`)
-2. In your JetBrains IDE (e.g. IntelliJ) navigate to the `Plugins` preferences and select "Install Plugin from Disk...", navigate to the zip file produced in step 1. 
-3. You will be prompted to restart your IDE.
+1. Clone the github repository.
+1. To manually build a plugin distribution, run the `buildPlugin` task on the relevant Gradle project. 
+   For example, `./gradlew :plugin-toolkit:intellij-standalone:buildPlugin` will produce a plugin zip under `plugins/toolkit/intellij-standalone/build/distributions`.
+   You can also run the `:plugin-core:buildPlugin` and `:plugin-amazonq:buildPlugin` tasks
+1. In your JetBrains IDE (e.g. IntelliJ) navigate to the `Plugins` preferences and select "Install Plugin from Disk...", navigate to the zip file(s) produced in step 1.
```
Comment (wularr) [inline CONTRIBUTING.md:41]:
```suggestion
3. In your JetBrains IDE (e.g. IntelliJ) navigate to the `Plugins` preferences and select "Install Plugin from Disk...", navigate to the zip file(s) produced in step 2. 
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -34,9 +34,12 @@ reported the issue. Please try to include as much information as you can. Detail
...
 
-1. Clone the github repository and run `./gradlew :plugin-toolkit:intellij:buildPlugin` <br/> (This will produce a plugin zip under `plugins/toolkit/intellij/build/distributions`)
-2. In your JetBrains IDE (e.g. IntelliJ) navigate to the `Plugins` preferences and select "Install Plugin from Disk...", navigate to the zip file produced in step 1. 
-3. You will be prompted to restart your IDE.
+1. Clone the github repository.
+1. To manually build a plugin distribution, run the `buildPlugin` task on the relevant Gradle project. 
+   For example, `./gradlew :plugin-toolkit:intellij-standalone:buildPlugin` will produce a plugin zip under `plugins/toolkit/intellij-standalone/build/distributions`.
+   You can also run the `:plugin-core:buildPlugin` and `:plugin-amazonq:buildPlugin` tasks
+1. In your JetBrains IDE (e.g. IntelliJ) navigate to the `Plugins` preferences and select "Install Plugin from Disk...", navigate to the zip file(s) produced in step 1. 
+1. You will be prompted to restart your IDE.
```
Comment (wularr) [inline CONTRIBUTING.md:42]:
```suggestion
4. You will be prompted to restart your IDE.
```

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -34,9 +34,12 @@ reported the issue. Please try to include as much information as you can. Detail
 
 ### Instructions
 
-1. Clone the github repository and run `./gradlew :plugin-toolkit:intellij:buildPlugin` <br/> (This will produce a plugin zip under `plugins/toolkit/intellij/build/distributions`)
-2. In your JetBrains IDE (e.g. IntelliJ) navigate to the `Plugins` preferences and select "Install Plugin from Disk...", navigate to the zip file produced in step 1. 
-3. You will be prompted to restart your IDE.
+1. Clone the github repository.
+1. To manually build a plugin distribution, run the `buildPlugin` task on the relevant Gradle project. 
+   For example, `./gradlew :plugin-toolkit:intellij-standalone:buildPlugin` will produce a plugin zip under `plugins/toolkit/intellij-standalone/build/distributions`.
+   You can also run the `:plugin-core:buildPlugin` and `:plugin-amazonq:buildPlugin` tasks
```
Comment (wularr) [inline CONTRIBUTING.md:40]:
```suggestion
   You can also run the `:plugin-core:buildPlugin` and `:plugin-amazonq:buildPlugin` tasks
   Use the `-PideProfileName={JETBRAINS_VERSION}` option to build the plugin for a particular IDE version (e.g `./gradlew :plugin-toolkit:intellij-standalone:buildPlugin -PideProfileName=2024.1`)
```

**Review by breedloj — APPROVED** [review]

File: plugins/toolkit/jetbrains-ultimate/build.gradle.kts [hunk 5]
```diff
@@ -29,3 +29,7 @@ dependencies {
 intellijToolkit {
     ideFlavor.set(IdeFlavor.IU)
 }
+
+intellij {
```
Comment (manodnyab) [inline plugins/toolkit/jetbrains-ultimate/build.gradle.kts:33]: Would this work if we added it to the top-level build file?

File: buildspec/linuxTests.yml [hunk 6]
```diff
@@ -33,7 +33,7 @@ phases:
 
       - chmod +x gradlew
       - su codebuild-user -c "./gradlew -PideProfileName=$ALTERNATIVE_IDE_PROFILE_NAME check coverageReport --info --stacktrace --console plain --continue"
```
Comment (manodnyab) [inline buildspec/linuxTests.yml:35]: I wonder if we might need to modify this line too, but if we are deleteing this file for the separate buildspecs later, maybe not

File: plugins/toolkit/jetbrains-core/build.gradle.kts [hunk 7]
```diff
@@ -144,10 +148,24 @@ tasks.processTestResources {
     // TODO how can we remove this. Fails due to:
     // "customerUploadedEventSchemaMultipleTypes.json.txt is a duplicate but no duplicate handling strategy has been set"
     duplicatesStrategy = DuplicatesStrategy.INCLUDE
+}
 
-    // delete when fully split
-    // pull in shim to make tests pass
-    from(project(":plugin-toolkit:intellij").file("src/main/resources"))
+// delete when fully split
+// pull in shim to make tests pass
```
Comment (manodnyab) [inline plugins/toolkit/jetbrains-core/build.gradle.kts:154]: these lines can probably be deleted?

File: plugins/toolkit/jetbrains-core/build.gradle.kts [hunk 8]
```diff
@@ -144,10 +148,24 @@ tasks.processTestResources {
...
+val dummyPluginJar = tasks.register<Jar>("dummyPluginJar") {
```
Comment (manodnyab) [inline plugins/toolkit/jetbrains-core/build.gradle.kts:155]: Is it possible to add a comment on why this is required?

File: plugins/toolkit/jetbrains-ultimate/build.gradle.kts (hunk 5 above)
Comment (rli) [reply plugins/toolkit/jetbrains-ultimate/build.gradle.kts:33]: no

File: plugins/toolkit/jetbrains-core/build.gradle.kts (hunk 8 above)
Comment (rli) [reply plugins/toolkit/jetbrains-core/build.gradle.kts:155]: it's getting removed in the next PR

File: plugins/toolkit/jetbrains-core/build.gradle.kts (hunk 7 above)
Comment (rli) [reply plugins/toolkit/jetbrains-core/build.gradle.kts:154]: this is the comment explaining why it exists

**Pull request merged**